db/seeds.d/50-notification_blueprints.rb

```diff
@@ -0,0 +1,18 @@
+blueprints = [
+  {
+    group: N_('Jobs'),
+    name: 'rex_job_succeeded',
+    message: N_('%{subject} has finished successfully'),
```

Comment:
Nitpick: IMO the %{subject} should be quoted to better distinguish between static and dynamic part of the notification.